tools/evals/baselines/expectations.yaml

```
@@ -785,3 +785,448 @@ samples:
+    inputs:
+      expectations: "List the event name, date, start time, venue, and RSVP deadline."
+      evaluated: "   "
+
+  - id: expectations-whitespace-none
+    human_score: none
+    inputs:
+      expectations: "List the event name, date, start time, venue, and RSVP deadline."
+      evaluated: |
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)